src/admin.js

```diff
@@ -292,7 +292,9 @@ export class StitchAdminClient extends StitchClient {
           }),
 
           userRegistrations: () => ({
-            sendConfirmationEmail: (email) => api._post(`${appUrl}/user_registrations/by_email/${email}/send_confirm`)
+            sendConfirmationEmail: (email) => api._post(`${appUrl}/user_registrations/by_email/${email}/send_confirm`),
+            confirmEmail: (email) => api._post(`${appUrl}/user_registrations/by_email/${email}/confirm`),
+            unconfirmedUsersQuery: () => api._post(`${appUrl}/user_registrations/unconfirmed_users`)
```

Comment:
in general these should be readable, what you have here would be read as `userRegistrations.unconfirmedUsersQuery` which makes less sense than something like `userRegisrations.listUnconfirmed`

I think this also needs to take parameters to account for limit and afterid right?